google/cloud/ndb/tasklets.py

```diff
@@ -25,17 +25,35 @@
 the tasklet, any yield of a Future waits for and returns the Future's result.
 For example::
 
+    from tasklets import tasklet, Return
+
     @tasklet
     def foo():
-        a = yield <some Future>
-        b = yield <another Future>
-        return a + b
+        a = yield <AFuture>
+        b = yield <BFuture>
+        raise Return(a + b)
```

Comment:
I don't want to be overemphasizing the Python 2 way of doing this, since part of the point of this package is to allow people to migrate to Python 3 so we can stop supporting Python 2. Maybe something like?

`        return a + b   # Python 2: raise Return(a + b)`

or

`        raise Return(a + b)  # Python 3: return a + b`

?

Reply:
I reworded based on your comments. See what you think.